application/src/main/java/org/opentripplanner/standalone/server/DefaultServerRequestContext.java

```diff
@@ -35,130 +35,98 @@
-    TileRendererManager tileRendererManager,
-    VectorTileConfig vectorTileConfig,
-    WorldEnvelopeService worldEnvelopeService,
     RealtimeVehicleService realtimeVehicleService,
-    VehicleRentalService vehicleRentalService,
+    List<RideHailingService> rideHailingServices,
+    RouteRequest routeRequestDefaults,
+    StreetLimitationParametersService streetLimitationParametersService,
+    TransitRoutingConfig transitRoutingConfig,
+    TransitService transitService,
+    VectorTileConfig vectorTileConfig,
     VehicleParkingService vehicleParkingService,
+    VehicleRentalService vehicleRentalService,
+    WorldEnvelopeService worldEnvelopeService,
     @Nullable EmissionsService emissionsService,
+    @Nullable LuceneIndex luceneIndex,
     @Nullable SorlandsbanenNorwayService sorlandsbanenService,
-    List<RideHailingService> rideHailingServices,
     @Nullable StopConsolidationService stopConsolidationService,
-    StreetLimitationParametersService streetLimitationParametersService,
-    FlexParameters flexParameters,
-    @Nullable TraverseVisitor traverseVisitor,
-    @Nullable LuceneIndex luceneIndex,
-    DebugUiConfig debugUiConfig
+    @Nullable TileRendererManager tileRendererManager,
+    @Nullable TraverseVisitor traverseVisitor
   ) {
+    this.debugUiConfig = debugUiConfig;
+    this.flexParameters = flexParameters;
     this.graph = graph;
-    this.transitService = transitService;
-    this.transitRoutingConfig = transitRoutingConfig;
     this.meterRegistry = meterRegistry;
     this.raptorConfig = raptorConfig;
-    this.tileRendererManager = tileRendererManager;
+    this.realtimeVehicleService = realtimeVehicleService;
+    this.rideHailingServices = rideHailingServices;
+    this.routeRequestDefaults = routeRequestDefaults;
+    this.streetLimitationParametersService = streetLimitationParametersService;
+    this.transitRoutingConfig = transitRoutingConfig;
+    this.transitService = transitService;
     this.vectorTileConfig = vectorTileConfig;
-    this.vehicleRentalService = vehicleRentalService;
     this.vehicleParkingService = vehicleParkingService;
-    this.flexParameters = flexParameters;
-    this.traverseVisitor = traverseVisitor;
-    this.routeRequestDefaults = routeRequestDefaults;
+    this.vehicleRentalService = vehicleRentalService;
     this.worldEnvelopeService = worldEnvelopeService;
-    this.realtimeVehicleService = realtimeVehicleService;
-    this.rideHailingServices = rideHailingServices;
     this.emissionsService = emissionsService;
+    this.luceneIndex = luceneIndex;
     this.sorlandsbanenService = sorlandsbanenService;
     this.stopConsolidationService = stopConsolidationService;
-    this.streetLimitationParametersService = streetLimitationParametersService;
-    this.luceneIndex = luceneIndex;
-    this.debugUiConfig = debugUiConfig;
-  }
-
-  /**
-   * Create a server context valid for one http request only!
-   */
-  public static DefaultServerRequestContext create(
```

Comment:
This more or less is a duplicate of the constructor - no need to repeat the same code twice.